src/components/Banner/index.tsx

```diff
@@ -28,8 +25,53 @@ const Banner = ({
         </a>
         {text}
       </p>
-    </div>
-  ) : null;
+    );
+  }
+
+  return null;
+};
+
+const useHtmlContent = ({ html, link }: BannersIndex) => {
+  if (html) {
+    // @note this is a workaround and it would be better if nodejs.org
+    // repository would provide absolute urls instead of relative for us
+    const replaceLocalLinksToNodejsOrg = html.replace(
+      "'/static/",
+      "'http://nodejs.org/static/"
+    );
+
+    return (
+      <a
+        href={isAbsoluteUrl(link) ? link : `http://nodejs.org/${link}`}
```

Comment:
```suggestion
        href={isAbsoluteUrl(link) ? link : `https://nodejs.org/${link}`}
```

It would be good to use HTTPS wherever possible (there's a few other references to HTTP too). Seeing `Mixed Content: The page at 'https://staging.nodejs.dev/2618/' was loaded over HTTPS, but requested an insecure element` on the preview

Reply:
This was a temporary workaround and can already be removed. (As the PR on nodejs.oeg was merged) And yes, definitely https is the way to go (we don't want to load mixed content here).

Reply:
Going to remove this snippet!